framework/src/st_things/things_stack/Makefile

```diff
@@ -0,0 +1,194 @@
+############################################################################
+# apps/dawit/Makefile
```

Comment:
1. path
2. samsung license
3. what is purpose of this file?

Reply:
remove this file